core/templates/pages/blog-home-page/blog-home-page.component.html

```diff
@@ -86,7 +86,7 @@
         <picture>
           <source type="image/webp" [srcset]="getStaticImageUrl('/images/general/no_explorations_found.webp')">
           <source type="image/png" [srcset]="getStaticImageUrl('/images/general/no_explorations_found.png')">
-          <img [src]="getStaticImageUrl('/general/no_explorations_found.png')">
+          <img  [src]="getStaticImageUrl('/general/no_explorations_found.png')">
```

Comment:
Drop the extra space before [src].

In general please look at the Files Changed tab before asking for a review, and ensure that the changes there correctly reflect the ones you want to make. Thanks.

Reply:
Sorry for that , I will  always keep it in mind.